### DISCUSSION TITLE
Provide directus v9 as docker image

### DISCUSSION THREAD

**Discussion by bitbaggi** [directus/directus#2847] [Archive]
we have an directus v8 instance on kubernetes with docker.

Now we want to try the version 9 of directus.

Could you provide this version as docker image?

**Comment by benhaynes** [comment]
I think we can start providing a Docker image for v9 once we hit Beta later this week... right @WoLfulus ?

**Comment by WoLfulus** [comment]
Already available in docker hub and GHCR
